src/main/java/UserInterface.java

```diff
@@ -2,96 +2,116 @@
 
 public class UserInterface {
     private static final String HORIZONTAL_LINE = "____________________________________________________________";
-
     private static final String LOGO = "\n" +
             "______  ___ ________  ___ ___  _   _ \n" +
             "| ___ \\/ _ |_   _|  \\/  |/ _ \\| \\ | |\n" +
             "| |_/ / /_\\ \\| | | .  . / /_\\ |  \\| |\n" +
             "| ___ |  _  || | | |\\/| |  _  | . ` |\n" +
             "| |_/ | | | || | | |  | | | | | |\\  |\n" +
             "\\____/\\_| |_/\\_/ \\_|  |_\\_| |_\\_| \\_/";
+    private static final int USER_COMMAND_INDEX = 0;
+    private static final int REMAINING_USER_INPUT_INDEX = 1;
+    private static final Scanner sc = new Scanner(System.in);
+    private static String[] userInputs;
 
-    private static final int FIRST_WORD = 0;
-    private static final int SECOND_WORD = 1;
-
-    private static void printLine() {
-        System.out.println(HORIZONTAL_LINE);
-    }
-
-    public static void greet() {
-        System.out.println("Greetings from\n" + LOGO);
-        printLine();
-        System.out.print("Hello! I'm BATMAN\n" + "What can I do for you?\n");
-        printLine();
+    public static void executeCommand(Command userCommand) {
+        switch (userCommand) {
+        case EXIT:
+            showExitMessage();
+            Duke.isRunning = false;
+            break;
+            
+        case LIST:
+            showList();
+            break;
+            
+        case ADD_TODO:
+        case ADD_DEADLINE:
+        case ADD_EVENT:
+            addTask(userCommand, userInputs[REMAINING_USER_INPUT_INDEX]);
+            break;
+            
+        case DONE:
+            int taskIndex = getTaskIndex(userInputs[REMAINING_USER_INPUT_INDEX]);
+            if (TaskManager.setDone(taskIndex)) {
+                showItemSetDone(taskIndex);
+                break;
+            }
+            showInvalidIndex();
+            break;
+            
+        default:
+            showInvalidCommand();
+            break;
+        }
     }
```

Comment:
Is there a need for spaces between cases?

Reply:
You are right I think there is no need for a space between cases. Thank you.